mParticle-Kochava/MPKitKochava.m

```diff
@@ -255,10 +259,40 @@ - (MPKitExecStatus *)didFinishLaunchingWithConfiguration:(NSDictionary *)configu
         KVALog.shared.level = [self.configuration[kvEnableLogging] boolValue] ? KVALogLevel.debug : KVALogLevel.never;
     }
     
-    if ([configuration[kvUseCustomerId] boolValue] || [configuration[kvIncludeOtherUserIds] boolValue]) {
+    if ([self.configuration[kvUseCustomerId] boolValue] || [self.configuration[kvIncludeOtherUserIds] boolValue]) {
         [self synchronize];
     }
     
+    NSDictionary *userActivityDictionary = self.launchOptions[UIApplicationLaunchOptionsUserActivityDictionaryKey];
+    if (userActivityDictionary == nil)
+    {
+        [KVADeeplink processWithURL:nil completionHandler:^(KVADeeplink * _Nonnull deeplink)
+         {
+            if (!deeplink) {
```

Comment:
Based on the code example screenshot in your PR and the `_Nonnull` specifier of the `completionHandler` parameter, this check will never be true, the check should actually be `if (deeplink.destinationString.length == 0` instead.

Reply:
Nice catch! I'll update the other call that was doing it this way as well.